src/genie/libs/parser/iosxe/show_bgp.py

```diff
@@ -5208,6 +5221,13 @@ def cli(self, neighbor, address_family='', output=None):
 
         return route_dict
 
+    def check_number_of_prefixes(self, output):
+        number_of_prefixes = re.compile(r'Total\s+number\s+of\s+prefixes\s+(?P<number_of_prefixes>\d+)\s*')
+        m = re.search(number_of_prefixes, output)
+        if not m:
```

Comment:
Does this line make sense? I'm not sure if this is the wanted behavior for this.

Reply:
Would likely be better to do 
```python
    def check_number_of_prefixes(self, output):
        number_of_prefixes = re.compile(r'Total\s+number\s+of\s+prefixes\s+(?P<number_of_prefixes>\d+)\s*')
        m = number_of_prefixes.search(output)
        if not m:
```
but the 
```python
if not m:
```
is fine